addons/storyshots/storyshots-core/src/api/snapshotsTestsTemplate.js

```diff
@@ -3,12 +3,13 @@ import { describe, it } from 'global';
 function snapshotTest({ story, kind, fileName, framework, testMethod, testMethodParams }) {
   const { name } = story;
 
-  it(name, () => {
+  it(name, (done) => {
```

Comment:
As far as I understand, if `done` is passed to the test function, jest recognizes it as an async test, and waits for the `done` to be invoked. As you can see all the tests failed because `done` was not invoked...

Reply:
thanks for your review @igor-dv  ... I'll fix this implementation ASAP

Reply:
@hisapy are you able to finish this PR?

Reply:
Sorry, I've been a little bit busy in some backend stuff ... I'll work on this next week